vendor/izinto/izi-io-4840-grid.yaml

```diff
@@ -0,0 +1,126 @@
+name: izi-io 4840 /grid
```

Comment:
@Kater--S can you put another name, this will be show on our device repository page, it will be better for users 

https://www.thethingsnetwork.org/device-repository/